step_definitions/steps.js

```diff
@@ -0,0 +1,22 @@
+const { I, janusObituaryPage } = inject();
+// Add in your custom step files
+
+Given(/I am on a (\w+) obituary page/, async (pageType) => {
+  console.log(pageType)
```

Comment:
If you need some long-term debugging output, you can use `I.say(<message>)`. The linter will complain about console.log